test/runtests.jl

```diff
@@ -1,7 +1,9 @@
+using Base: sign_mask
```

Comment:
Not sure what this is.

Reply:
Sorry about this. One of my VSCode plugins adds spurious declarations. I have not been able to figure out which one.